### DISCUSSION TITLE
Is there any way to extract the scene information in a sentence using spaCy?

### DISCUSSION THREAD

**Discussion by tiesanguaixia** [explosion/spaCy#13182] [Help: Coding & Implementations]
For example, a sentence is "A man is cooking in **kitchen**." Is spaCy or any other NLP tools can extract the scene information like "kitchen"?

**Comment by rmitsch** [comment]
Hi @tiesanguaixia, if I understand this correctly, you want to extract locations in a general sense, right? This sounds like an [NER task](https://spacy.io/usage/linguistic-features#named-entities). spaCy's pretrained models like `en_web_lg` support NER with a LOCATION tag, this might work well enough for you.

Otherwise I recommend looking into training your own model with exactly those tags you are interested in.
